SMPost/schema.json

```diff
@@ -1,98 +1,158 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "schemaVersion": "0.0",
-  "$id": "https://raw.githubusercontent.com/smart-data-models/dataModel.SocialMedia/master/SMPost/schema.json",
+  "$id": "https://raw.githubusercontent.com/satzanakis/dataModel.SocialMedia/master/SMPost/schema.json",

```

Comment:
the original one was right (when approved this will be the url)

Reply:
My bad, testing, fixed.